assets/css/admin.scss

```
@@ -81,6 +80,56 @@
 		}
 	}
 
+	.top-bar {
+		background: #fff;
+		box-shadow: inset 0 -1px 0 #ccc;
+		display: block;
+		margin: 0 0 16px;
+
+		@media only screen and ( min-width: 768px ) {
+			margin-bottom: 24px;
+		}
+
+		ul {
+			background: #fff;
+			box-shadow: inset 0 -1px 0 #ccc;
+			display: flex;
+			justify-content: left;
+			margin: 0 auto;
+			max-width: 1056px;
+			width: 100%;
+			overflow-x: scroll;
+
+			@media only screen and ( min-width: 1100px ) {
+				justify-content: center;
+			}
+
+			li {
+				font-weight: 600;
+				font-size: 14px;
+				line-height: 20px;
+				margin: 0;
+			}
+
+			a,
+			a:visited,
+			a:hover,
+			a:focus {
+				border: none;
+				color: #1e1e1e;
+				display: inline-block;
+				text-decoration: none;
+				outline: none;
+				padding: 20px 16px;
```

As a temporary measure, to let the bar fit on 1024:

```suggestion
				padding: 20px 12px;
```

---

Fixed it.